assets/xml/overlays/ovl_Boss_Ganon2.xml

```diff
@@ -1,77 +1,29 @@
-        </Array>
-        <Array Name="ovl_Boss_Ganon2_Vtx_010038" Count="12" Offset="0x010038">
-            <Vtx/>
-        </Array>
-        <Array Name="ovl_Boss_Ganon2_Vtx_0100F8" Count="10" Offset="0x0100F8">
-            <Vtx/>
-        </Array>
-        <Array Name="ovl_Boss_Ganon2_Vtx_010198" Count="16" Offset="0x010198">
-            <Vtx/>
-        </Array>
-        <Array Name="ovl_Boss_Ganon2_Vtx_010298" Count="17" Offset="0x010298">
-            <Vtx/>
-        </Array>
-        <DList Name="ovl_Boss_Ganon2_DL_0103A8" Offset="0x0103A8"/>
+        <Texture Name="gGanonLightOrbTex" OutName="ganon_light_orb" Format="i8" Width="64" Height="64" Offset="0x00A2C8"/>
+        <DList Name="gGanonLightOrbMaterialDL" Offset="0x00B308"/>
+        <DList Name="gGanonLightOrbModelDL" Offset="0x00B378"/>
+        <DList Name="gGanonShadowMaterialDL" Offset="0x00B3D0"/>
+        <DList Name="gGanonShadowModelDL" Offset="0x00B3F0"/>
+        <Texture Name="gGanonSwordTrailTex" OutName="ganon_sword_trail" Format="i8" Width="32" Height="32" Offset="0x00B420"/>
+        <Texture Name="gGanonSwordTrailMaskTex" OutName="ganon_sword_trail_mask" Format="i4" Width="32" Height="32" Offset="0x00B820"/>
+        <Array Name="gGanonSwordTrailVtx" Count="22" Offset="0x00BA20">
+            <Vtx/>
+        </Array>
+        <DList Name="gGanonSwordTrailDL" Offset="0x00BB80"/>
+        <Texture Name="gGanonTriforceTex" OutName="ganon_triforce" Format="i8" Width="64" Height="64" Offset="0x00BC98"/>
+        <DList Name="gGanonTriforceDL" Offset="0x00CCD8"/>
+        <Texture Name="gGanonLightningTex" OutName="ganon_lightning" Format="i4" Width="32" Height="160" Offset="0x00CD58"/>
+        <DList Name="gGanonLightningDL" Offset="0x00D798"/> <!-- Original name might be "efc_fg2_thunder1_modelT" -->
+        <Texture Name="gGanonFireRingTex" OutName="ganon_fire_ring" Format="i8" Width="32" Height="64" Offset="0x00D820"/>
+        <DList Name="gGanonFireRingDL" Offset="0x00E1C0"/> <!-- Original name is "ganon_fire_modelT" -->
+        <Texture Name="gGanonZeldaMagicTex" OutName="ganon_zelda_magic" Format="i8" Width="32" Height="64" Offset="0x00E2E0"/>
+        <DList Name="gGanonZeldaMagicDL" Offset="0x00EC40"/> <!-- Original name is "efc_ganon2_hadou_modelT" ("wave motion; undulation; surge​") -->
+        <Texture Name="gGanonMasterSwordShadowTex" OutName="ganon_master_sword_shadow" Format="i8" Width="32" Height="32" Offset="0x00ED48"/>
+        <DList Name="gGanonMasterSwordShadowDL" Offset="0x00F188"/> <!-- Original name is "master_sword_shadow_model" -->
+        <Texture Name="gGanonMasterSwordPommelTex" OutName="ganon_master_sword_pommel" Format="rgba16" Width="16" Height="16" Offset="0x00F208"/>
+        <Texture Name="gGanonMasterSwordGuardTex" OutName="ganon_master_sword_guard" Format="rgba16" Width="32" Height="32" Offset="0x00F408"/>
+        <DList Name="gGanonMasterSwordDL" Offset="0x0103A8"/> <!-- Original name is "master_gn2_swordB_model" -->
```

Comment:
```suggestion
        <Texture Name="gGanonMasterSwordGuardTex" OutName="ganon_master_sword_guard" Format="rgba16" Width="32" Height="32" Offset="0xF408"/>
        <DList Name="gGanonMasterSwordDL" Offset="0x103A8"/> <!-- Original name is "master_gn2_swordB_model" -->
```
Just the smallest nit. Could you remove the leading zero's from the offset to clean it up?